<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogConfirm</class>
 <widget class="QDialog" name="dialogConfirm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirm Resolution</string>
  </property>
  <property name="windowIconText">
   <string/>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="labelText">
     <property name="text">
      <string>Do you wish to use this resolution?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="textAutoClose">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushYes">
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushNo">
     <property name="text">
      <string>No</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>71</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushNo</tabstop>
  <tabstop>pushYes</tabstop>
 </tabstops>
 <includes>
  <include location="local">qtimer.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>pushYes</sender>
   <signal>clicked()</signal>
   <receiver>dialogConfirm</receiver>
   <slot>slotYes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushNo</sender>
   <signal>clicked()</signal>
   <receiver>dialogConfirm</receiver>
   <slot>slotNo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
